src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star stars__star-active"></div>
+        <div class="stars__star"></div>
```

Comment:
This is not the approach we want in this case.
Star should have only one class and you should change background-image basing on parent's class (in this case `stars--4`. You can use `:nth` pseudo class to make it work 😸)

Btw modifiers should have two dashes ;)